metrics/web/docs/features/firestore_emulator_for_driver_tests/diagrams/firestore_emulator_sequence_diagram.puml

```diff
@@ -0,0 +1,42 @@
+@startuml firestore_emulator_sequence_diagram
```

Comment:
Would it be great to add an application side to this diagram and explain the process of configuring the Firestore instance to use the emulator?